INFO _internal.py:88  * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:38] "GET / HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:44] "GET /api/v1_0 HTTP/1.1" 301 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:44] "GET /api/v1_0/ HTTP/1.1" 500 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:44:45] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/api_v_1_0/index.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:49] "GET /api/v1_0/ HTTP/1.1" 500 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:49] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:49] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:49] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:49] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:50] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:46:50] "GET /api/v1_0/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/api_v_1_0/index.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO index.py:12 hshsh
INFO _internal.py:88 127.0.0.1 - - [09/Nov/2018 09:48:17] "GET /api/v1_0/ HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88  * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
ERROR __init__.py:40 000000000
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
ERROR __init__.py:40 000000000
INFO _internal.py:88  * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
ERROR __init__.py:40 000000000
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
INFO _internal.py:88  * Detected change in '/Users/android-school/Python/PythonStudy/Flask/Flask-Home3/ihome/__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
ERROR __init__.py:40 000000000
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
ERROR __init__.py:40 000000000
INFO _internal.py:88  * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
ERROR __init__.py:40 000000000
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 647-713-143
